projects/lights_control/src/lights_config.c

```diff
@@ -0,0 +1,21 @@
+#include "lights_config.h"
+#include "can_msg_defs.h"
+#include "lights_can.h"
+
+const CANSettings can_settings_front = { .bitrate = CAN_HW_BITRATE_125KBPS,
```

Comment:
These are the same you could just populate the correct device_id field once you determine what board you are? Consider a lights_config function `lights_config_set_board_type`?